_pages/how-to-submit.md

```diff
@@ -0,0 +1,213 @@
+
+> New to Markdown? Here's a great [Markdown Guide](https://www.markdownguide.org/basic-syntax/) that covers all of the basic syntax.
+
+## <i class="fa-solid fa-map"></i> Step-by-step guide to completing the pyOpenSci software review issue template
+
+When completing your package submission, it's best to submit the issue all at once, rather than completing it over the course of several sessions. We recommend reading through the issue once or twice to see what information you'll need, which includes:
+
+* The scope of your package (see categories)
+* Your target audience
+* Scientific applications of your package
+* Other packages that accomplish the same thing (if any), and how yours is different
+
+> Remember: keep the formatting of the issue template exactly as it is! It's OK to leave sections that don't apply to you blank, and there's no need to add any additional formatting (bolding, italicizing, etc.). Any of these types of changes make it more difficult for our editorial team to process your package submission.
+
+### Package metadata
+
+* **Add a title:** keep your title simple. Oftentimes your package name and the word submission is enough, such as "My Amazing Python package submission."
+* **Add a description** in this section you'll want to fill out the Submitting Author, All current maintainers, Package Name, One-Line Description of Package, Repository Link, and Version submitted sections. You should *not* fill out the EiC, Editor, Reviewer 1, Reviewer 2, Archive, JOSS DOI, Version accepted, or Date accepted sections. Instead, leave these as is, with "TBD."
+
+### Code of Conduct & Commitment to Maintain Package
+
+In order for your package to progress through the pyOpenSci review process, you'll need to check both of the [Code of Conduct](https://www.pyopensci.org/handbook/CODE_OF_CONDUCT.html) agreement as well as the [maintenance agreement](https://www.pyopensci.org/software-peer-review/our-process/policies.html#after-acceptance-package-ownership-and-maintenance) outlined in the [pyOpenSci Policies Guidelines](https://www.pyopensci.org/software-peer-review/about/intro.html). You can indicate your agreement in one of two ways:
+* adding an "X" (without quotation marks) inside the brackets in the Markdown version of the issue, so that it reads like this: [X]
+* submitting the issue without making any changes to this section, and then clicking on the checkboxes when they're rendered in the submitted issue.
+
+### Description
+
+This section is where you'll provide a brief description of your package. It's helpful to include information on what your package does, and how it helps to benefit the open science community. Sharing information about what your package allows users to do helps us get a clearer idea on the scope of your package, as well as help us identify editors and reviewers who will be a good fit for your review.
+
+### Scope
+
+The mission of pyOpenSci’s open peer review process is to:
```

Comment:
i think this section should be simpler and the context / mission related stuff would be better fit higher up on this page. here you can just tell the user to select the scope categories that best align with their tools goals. They can pick one or more but if it's data viz they should submit a pre-submission inquiry.  

For the how-to around this scope section, I would not copy content from the template because it will become quickly dated if we enhance the scope. You could provide a few examples and link to the template for all of the items.